info/about.html

```diff
@@ -0,0 +1,41 @@
+<html>
+	<head>
+		<title>About Me</title>
+		<link rel="stylesheet" href="style.css">
+	</head>
+
+<body>
+	<nav>
+		<a href="about.html">About Me</a> |
+		<a href="portfolio">Portfolio</a>
+	</nav>
+
+	<h2>TAYLOR HIDEN</h2>
+
+	<br></br>
+
```

Comment:
There's no end tag for br. Also, some devs can be snotty about using the br tag-they think it's bad form. Take a look here to learn more: http://line25.com/articles/10-html-tag-crimes-you-really-shouldnt-commit